phc-rails/app/models/service_list.rb

```diff
@@ -0,0 +1,16 @@
+# This table has a single row for each unique service
+
+class ServiceList < ActiveRecord::Base
+    def self.services
+      self.all.map(&:salesforce_name)
+      # services = ["Acupuncture__c", "Addiction_Recovery__c", "Adult_Probation__c", "Banking__c", \
+      #             "Books__c", "CAAP__c", "CalFresh__c", "Dental__c", "Disability_Services__c", \
+      #             "DMV_ID__c", "Employment__c", "Family_Services__c", "Flu_Shot__c", "Foot_Washing__c", \
+      #             "Foodbank__c", "Haircuts__c", "HIV_STI_Testing__c", "Homeward_Bound__c", \
+      #             "Housing_Info__c", "Legal__c", "Lifeline_Cell__c", "Lunch__c", "Massage__c", \
+      #             "Medical__c", "SSI_SSDI_Medi_Cal__c", "Mental_Health__c", "Needle_Exchange__c", \
+      #             "Pet_Care__c", "Phone_Calls__c", "Photo_Portraits__c", "Vision_Readers__c", \
+      #             "Senior_Services__c", "SSI_SSDI_Medi_Cal__c", "TB_Testing__c", "Veteran_Services__c", \
+      #             "Vision_Screening__c", "Voter_Registration__c", "Wheelchair_Repair__c", "Youth_Services__c"]
+    end
+end
```

Comment:
just discussed this in person but we should have an `in_use` toggle to indicate whether the service is being offered for the current event. i foresee there eventually being ~50-60 services in this table, about 30 of which are activated for each event.

alternatively, we could also create a 'PHCEvent' model that has_many EventRegistrations and has_many ServiceLists. when they create the PHC event in admin, they could then select from a dropdown menu all the services that are offered at the event. pro of this would be that they wouldn't have to individually toggle the in_use every event. we could also start storing event registration data from more than one event.